static/scripts/navigation-loader.js

```diff
@@ -0,0 +1,261 @@
+/**
```

Comment:
Everything from `src/scripts` gets bundled into `static/scripts/main.js` on build, so I don't think this file is necessary